android/KMEA/app/src/main/java/com/tavultesoft/kmea/data/LexicalModel.java

```diff
@@ -45,14 +46,14 @@ public LexicalModel(JSONObject installedObj) {
       String packageID = "";
       if (lexicalModelJSON.has(KMManager.KMKey_PackageID)) {
         packageID = lexicalModelJSON.getString(KMManager.KMKey_PackageID);
-      } else if (kmp != null && FileUtils.hasLexicalModelPackageExtension(kmp)) {
-        // Extract package ID from packageFilename
-        String filename = FileUtils.getFilename(kmp);
-        // Truncate .model.kmp file extension
-        packageID = filename.replace(FileUtils.MODELPACKAGE, "");
+      } else if (kmp != null) {
```

Comment:
This is using the lexical-model catalog JSON from https://api.keyman-staging.com/model?q
so the `packageFilename` fields don't contain the redirect URLs.